Software/Python/grove_barometer_sensors/barometric_sensor_bmp180/Adafruit_I2C.py

```diff
@@ -1,4 +1,8 @@
 #!/usr/bin/python
+
```

Comment:
This is an Adafruit file. I'm not sure we should change it by adding a license. Unless Adafruit has it like that.